spec/lib/public_urls_spec.rb

```diff
@@ -1,9 +1,14 @@
 RSpec.describe PublicUrls do
   describe ".url_for" do
-    it "returns the GOV.UK url for the content item" do
+    it "returns a GOV.UK for a base path" do
       result = subject.url_for(base_path: "/foo/bar")
```

Comment:
it returns a GOV.UK url for a base path?

Reply:
Oops - I've updated the second commit with "URL" 👍.